### ISSUE TITLE
Server error when trying to reset website data

### ISSUE THREAD

**Issue by obrassard** [umami-software/umami#2061]
### Describe the Bug

I was trying to delete data from a specific website using the **Reset website** option, however after typing "RESET" in the form, I get a 500 error.

I pasted the logs from the docker container below, it seems to be a foreign key issue (no cascade) : 
 
<img width="494" alt="Capture d’écran, le 2023-05-29 à 18 37 36" src="https://github.com/umami-software/umami/assets/17227551/bb8c152d-cbbc-410f-9ae1-a59eff877338">

### Database

MySQL

### Relevant log output

```shell
Invalid `prisma.websiteEvent.deleteMany()` invocation:

The change you are trying to make would violate the required relation 'EventDataToWebsiteEvent' between the `EventData` and `WebsiteEvent` models.
    at pn.handleRequestError (/app/node_modules/@prisma/client/runtime/library.js:176:6477)
    at pn.handleAndLogRequestError (/app/node_modules/@prisma/client/runtime/library.js:176:5907)
    at pn.request (/app/node_modules/@prisma/client/runtime/library.js:176:5786)
    at async t._request (/app/node_modules/@prisma/client/runtime/library.js:179:10484) {
  code: 'P2014',
  clientVersion: '4.13.0',
  meta: {
    relation_name: 'EventDataToWebsiteEvent',
    model_a_name: 'EventData',
    model_b_name: 'WebsiteEvent'
  }
}
```

### How are you deploying your application? (if relevant)

Docker

**Comment by franciscao633** [comment]
Confirmed the bug., will fix next release.  Thanks for reporting.

**Comment by franciscao633** [comment]
fixed in v2.3.0
